g3web/WebContent/welcome.jsp

```diff
@@ -0,0 +1,5 @@
+<%@ taglib uri="/struts-tags" prefix="s"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<b> WELCOME USER </b>
+</html>
```

Comment:
I tried pressing cntrl+shift+f but it remained same. I will try to format it again when we build complete welcome page.